recipes/baked_beans.json

```
@@ -0,0 +1,18 @@
+{
+    "recipe_name": "Baked Beans",
+    "serves": "1",
+    "prep_time": "2 mins",
+    "cooking_time": "2 mins",
+    "ingredients": [
+        {
+            "unit_size": "1",
+            "unit_type": "tin",
+            "ingredient_name": "heinz baked beans"
+        }
+    ],
+    "method": [
+        "Empty tin of baked beans into a bowl.",
```

```suggestion
        "Empty tin of baked beans into a bowl, cover and vent.",
```

True microwave connoisseurs cover and vent their beans.